app/build.gradle

```diff
@@ -174,6 +174,7 @@ dependencies {
     implementation project(':autofill-api')
     implementation project(':autofill-impl')
     implementation project(':autofill-store')
+    internalImplementation project(':autofill-internal')
```

Comment:
Adds some dev settings for autofill and email protection (like clearing the "never ask again" choice)